ts/features/bonus/cgn/store/reducers/eyca/details.ts

```diff
@@ -45,3 +48,22 @@ export default reducer;
 
 export const eycaDetailSelector = (state: GlobalState): EycaDetailsState =>
   state.bonus.cgn.eyca.details;
+
+export const isEycaEligible = createSelector(
+  eycaDetailSelector,
+  (eycaDetail: EycaDetailsState): boolean =>
+    isReady(eycaDetail) && eycaDetail.value.status !== "INELIGIBLE"
+);
+
+export const eycaInformationSelector = createSelector(
```

Comment:
```suggestion
export const eycaCardSelector = createSelector(
```
Information could be misleading

Reply:
Fixed in 059777b